interfaces/__template__/v0/schema.py

```diff
@@ -12,5 +12,5 @@ class ProviderSchema(DataBagSchema):
     """The schema for the provider side of this interface."""
 
 
-class ConsumerSchema(DataBagSchema):
-    """The schema for the consumer side of this interface."""
+class RequirerSchema(DataBagSchema):
```

Comment:
I know you wrote "Additionally, this PR uniforms and enforces the official provider/requirer naming. (used to be: provider/consuming in some places). Hence the large diff!" but this should be a different PR. Please keep PR's scoped.

Reply:
Fair enough. See #61

Reply:
I rebased it on it, but couldn't change the merge target to have a clean diff. Assumption is we'll merge the other one first since at this point they're hard to disentangle.